CodingAChat/Scenes/Settings/SettingsCoordinator.swift

```diff
@@ -13,9 +13,11 @@ class SettingsCoordinator: Coordinator {
     var childCoordinators: [Coordinator] = []
     private let viewController: SettingsViewController
     private let presenter: UIViewController
+    private let tracker : Trackable
     
-    init(presenter: UIViewController) {
+    init(presenter: UIViewController, tracker : Trackable) {
         self.presenter = presenter
+        self.tracker=tracker
```

Comment:
assignment want spaces in both sides:
```suggestion
        self.tracker = tracker
```